fluent_comments/views.py

```diff
@@ -81,7 +81,7 @@ def post_comment_ajax(request, using=None):
     # Otherwise create the comment
     comment = form.get_comment_object()
     comment.ip_address = request.META.get("REMOTE_ADDR", None)
-    if request.user.is_authenticated():
+    if request.user.is_authenticated is True:
```

Comment:
This likely fails for Django versions that return `CallableTrue`. It likely needs some compat function instead.